smart_card_connector_app/src/window-devices-displaying.js

```diff
@@ -148,9 +176,12 @@ function getUserSelectedDevicesCallback(devices) {
 }
 
 GSC.ConnectorApp.Window.DevicesDisplaying.initialize = function() {
-  loadDeviceList();
```

Comment:
I think it's still useful to log the data supplied by the chrome.usb API, in case something in PC/SC or CCID went wrong.